docs/guides/airnode/post-processing/index.md

```diff
@@ -0,0 +1,256 @@
+will be calculated after calling the CoinGecko operation
+`/coins/{coin}/market_chart`.
+
+## 1. Setup a Docker container to run Airnode
+
+A local Docker container running Airnode is needed. Go to the guide
+[Deploying Airnode locally using Docker](/guides/airnode/deploy-airnode/deploy-container/index.md)
+to set one up. Once the container is running, return to step #2 below.
+
+## 2. Update the `config.json` file
+
+The project that was used for the Airnode deployed in step #1 above contains a
+`config.json` file. Rename it to something like `config-org.json` and create a
+new blank file named `config.json` in the same project folder. Then copy and
+paste the contents of the `config.json` file show below into your newly created
+file.
+
+::: details Copy/paste config.json
+
+```json
+<!--@include: ./src/config.json-->
+```
+
+:::
+
+The newer `config.json` file points to a different API provider operation
+(`/coins/{coin}/market_chart`).
+
+```js
+"paths": {
+  "/coins/{coin}/market_chart": {
+    "get": {
+      "parameters": [
+        ...
+      ]
+    }
+  }
+},
+```
+
+## 3. Review the `postProcessingSpecifications`
+
+After calling the API provider's operation but before encoding the resulting
+values for on-chain use, Airnode will run the post processing code defined in
+`postProcessingSpecifications`. During execution of this code, the data from the
+API provider operation can be summarized into the values needed.
+
+```js
+"preProcessingSpecifications": [],
+"postProcessingSpecifications": [
+  {
+    "environment": "Node",
+    "timeoutMs": 5000,
+    "value": "   let avg_price = 0;   let avg_market_cap = 0;   input.prices.forEach(el => { avg_price = avg_price + el[1]; });   input.avg_price = (avg_price / 7).toString();   input.market_caps.forEach(el => { avg_market_cap = avg_market_cap + el[1]; });   input.avg_market_cap = (avg_market_cap / 7).toString();   output = input;   "
+  }
+]
+```
+
+Below is a more human readable rendering of the post processing code. The
+`input` variable contains the response from the API provider's operation. The
+`output` variable must be set before the code exists as Airnode uses it to
+encode any values for use on-chain. See more about
+[Input and Output](/reference/ois/latest/processing.md#input-and-output) in the
+OIS reference.
+
+```js
+let avg_price = 0;
+let avg_market_cap = 0;
+input.prices.forEach((el) => {
+  avg_price = avg_price + el[1];
+});
+input.avg_price = (avg_price / 7).toString();
+input.market_caps.forEach((el) => {
+  avg_market_cap = avg_market_cap + el[1];
+});
+input.avg_market_cap = (avg_market_cap / 7).toString();
+output = input;
+```
+
+## 4. Remove the existing Docker container
+
+The Docker container you created in step #1 must be deleted and a new one
+created which uses the newer `config.jon` file. From within the Docker desktop
+app find the docker container named **quick-start-container-airnode** and delete
+it.
+
+<img src="./src/delete-container.png"/>
```

Comment:
I suggest deleting this step per my comment above.

Reply:
We can do this if the quick start tutorial uses the new configuration file.